src/sim_recon/cli/parsing.py

```diff
@@ -209,7 +210,6 @@ def parse_recon_args(
         namespace,
         ("-c/--config-path", "config_path"),
         ("-d/--data", "sim_data_paths"),
-        ("-o/--output-directory", "output_directory"),
```

Comment:
Why don't you just set the `required` argument?

Reply:
I'm parsing twice, so having it set as required means that passing `--help` will just throw an error about missing required arguments. Also, help needs to be added last (after the first parse) otherwise it won't be able to display all the arguments.

Reply:
I would have added all the options before parsing the arguments and wrote a helper function to combine the parsed arguments with the configured values.